Converter/JobsConverter.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/*
+ * This file is part of the Integrated package.
+ *
+ * (c) e-Active B.V. <integrated@e-active.nl>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Integrated\Bundle\ExportBundle\Converter;
+
+use Doctrine\ODM\MongoDB\DocumentManager;
+
+use Integrated\Bundle\ContentBundle\Document\Content\Relation\Company;
+use Integrated\Bundle\ContentBundle\Document\Content\Relation\Person;
+use Integrated\Common\ContentType\ContentTypeInterface;
+
+/**
+ * @author Jeroen van Leeuwen <jeroen@e-active.nl>
+ */
+class JobsConverter implements ConverterInterface
+{
+    /**
+     * @var DocumentManager
+     */
+    protected $dm;
+
+    /**
+     * @param DocumentManager $dm
+     */
+    public function __construct(DocumentManager $dm)
+    {
+        $this->dm = $dm;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function supports(ContentTypeInterface $contentType)
+    {
+        return is_subclass_of($contentType->getClass(), Person::class);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function convert(array $data)
+    {
+        $convert = [
+            'company' => new ConverterValue('job_1_companny', 'Job 1 company', ''),
```

Comment:
Wouldn't it be better to have the array key be the same as the object key. And in that case the key could actualy be removed from the object.